### PR TITLE
Translate CVE-2015-7551(zh-cn)

### PR DESCRIPTION
**Pull request by dsh0416**

### CODE REVIEW

**Comment by AndorChen** [comment]
Thanks for your contribution, but

1. The hard breaks within paragraphs just DON'T make sense for Chinese.
2. You use a lot "被". I would suggest changing the expression.

**Comment by dsh0416** [comment]
@AndorChen A new commit has been submitted to remove hard breaks and passive expressions.

File: zh_cn/news/_posts/2015-12-16-unsafe-tainted-string-usage-in-fiddle-and-dl-cve-2015-7551.md [hunk 1]
```diff
@@ -8,17 +8,13 @@ tags: security
...
-Fiddle 和 DL 的实现中使用了一个易受攻击的不安全的字符串调用。
-此风险已经被分配了 CVE 识别号
-[CVE-2015-7551](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2015-7551) 。
+Fiddle 和 DL 的实现中使用了一个易受攻击的不安全的字符串调用。[CVE-2015-7551](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2015-7551) 对此风险提供了识别号。
 
 ## 详情
 
-Fiddle 和 DL 的实现中使用了一个易受攻击的不安全的字符串调用。
-问题最早在有关于 DL 的 [CVE-2009-5147](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2009-5147) 中被报告并修复，但在 DL 使用 Fiddle 和 libffi 重构后再此出现。
+Fiddle 和 DL 的实现中使用了一个易受攻击的不安全的字符串调用。最早的报告和修复出现在有关于 DL 的 [CVE-2009-5147](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2009-5147) 中，但在 DL 使用 Fiddle 和 libffi 重构后再此出现。
```
Comment (AndorChen) [inline zh_cn/news/_posts/2015-12-16-unsafe-tainted-string-usage-in-fiddle-and-dl-cve-2015-7551.md]: 再此出现->再次出现?

**Comment by dsh0416** [comment]
@AndorChen Fixed in the latest commit

**Pull request merged**